demisto_sdk/commands/pre_commit/hooks/ruff.py

```diff
@@ -8,13 +10,19 @@ def _python_version_to_ruff(python_version: str):
 
     def prepare_hook(
         self,
-        python_version: str,
+        python_version_to_files: dict,
```

Comment:
```suggestion
        python_version_to_files: dict[str, Any],
```
Add the correct typing